internal/cli/cli.go

```diff
@@ -366,7 +366,7 @@ func shouldPromptWhenFlagless(cmd *cobra.Command, flag string) bool {
 	isSet := false
 
 	cmd.LocalFlags().VisitAll(func(f *pflag.Flag) {
-		if f.Changed {
+		if f.Name == flag && f.Changed {
```

Comment:
This visits all the flag and if any is set, it takes it in account for the return value. If we check for the name, aren't we only taking in account the status of that single flag?
Context: https://github.com/auth0/auth0-cli/issues/130#issuecomment-791683725

Reply:
Hmm, I see. Ok we can close this totally misunderstood the intent of this 👍